ux.symfony.com/assets/styles/demos/live-memory/app/_reset.scss

```diff
@@ -0,0 +1,34 @@
+// -----------------------------------------------------------------
+// Reset
+// -----------------------------------------------------------------
+
+*, *::before, *::after {
```

Comment:
Do we need this?

Reply:
I'm not using Bootstrap there, meaning my stylesheet file is 27kB ... when any other demo is 300kb :)

I had the intention of showing there how we could ditch bootrstrap next ;)